model/host/host.go

```diff
@@ -341,6 +342,27 @@ func (h *Host) SetDNSName(dnsName string) error {
 	return err
 }
 
+func (h *Host) SetIPv6Address(ipv6Address string) error {
+	err := UpdateOne(
+		bson.M{
+			IdKey:  h.Id,
+			IPv6Key: "",
+		},
+		bson.M{
+			"$set": bson.M{
+				IPv6Key: ipv6Address,
+			},
+		},
+	)
+	if err == nil {
+		h.IPv6 = ipv6Address
+	}
+	if err == mgo.ErrNotFound {
+		return nil
+	}
```

Comment:
arguably we shouldn't swallow err not found.

Reply:
I just grabbed this format from SetDNSName, where we also swallow err not found. Is there a reason we did that? If we change it here we should probably change it there.

Reply:
Evergreen has a long (and terrible history) of returning nil, nil from a lot of db functions which is super confusing, and I think we just have to draw a line somewhere.

Reply:
This will produce an "ErrorNotFound" in the case that the host exists but the IPv6 address for this host has already been set. I'm not sure if this case would even happen but to do it in one API call that would be the behavior anyway.